config/500-sources-controller.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         app: sources-controller
         eventing.knative.dev/release: devel
     spec:
-      serviceAccountName: eventing-controller
+      serviceAccountName: eventing-source-controller
```

Comment:
I like the separation here, @daisy-ycguo !

Reply:
thanks.